R/count_watershed_data.R

```diff
@@ -431,7 +452,8 @@ count_watershed_data <- function(data_dir,
               "runoff from developed land",      "%",        developed_runoff_percent,
               "total irrigation consumption",    "BCM",      total_irr_bcm,
               "population",                      "people",   wtrshd_population,
-              "population water consumption",    "ltr/sqkm", pop_water_consum
+              "population water consumption",    "ltr/sqkm", pop_water_consum,
+              "waste discharge per person",      "MGD/pers", wastedischarge_per_person
```

Comment:
Please convert from MGD to cumecs (m3/s) for consistency with the rest of the package, which is using metric units.